ERCS/erc-7786.md

```diff
@@ -74,6 +74,13 @@ interface IERC7786GatewaySource {
         bytes calldata payload,
         bytes[] calldata attributes
     ) external payable returns (bytes32 outboxId);
+
+    function requestRelay(
+        bytes32 outboxId,
+        uint256 gasLimit,
+        uint256 relayerFee,
+        string calldata refundReceiver
```

Comment:
why is this calldata as opposed to just an address?

Reply:
Because it's an address on the destination chain it has to be in CAIP-10 format or whatever format we end up using. It's not an EVM address necessarily.